src/AlgebraicGeometry/ToricVarieties/NormalToricVarieties/attributes.jl

```diff
@@ -210,7 +210,10 @@ julia> coordinate_names(antv)
 ```
 """
 @attr Vector{String} function coordinate_names(v::NormalToricVarietyType)
-    return ["x$(i)" for i in 1:torsion_free_rank(torusinvariant_weil_divisor_group(v))]
+  if has_attribute(v, :cox_ring)
+    return string.(gens(cox_ring(v)))
```

Comment:
```suggestion
    return string.(symbols(cox_ring(v)))
```
if you don't need the gens themselves but only their printing, this change avoids some allocations